src/property.h

```diff
@@ -145,8 +152,10 @@ struct Property : public detail::Property {
     PROPERTY(MOTORSTROM, 0x069f);
     PROPERTY(MOTORLEISTUNG, 0x06a0, Type::et_cent_val);
     PROPERTY(MOTORSPANNUNG, 0x06a1);
+    PROPERTY(OELSUMPFTEMPERATUR, 0x0693, Type::et_dec_val); 
     PROPERTY(ABLUFTTEMP, 0x0694, Type::et_dec_val);
-    PROPERTY(ANZEIGE_NIEDERDRUCK, 0x07a7, Type::et_dec_val);
+    PROPERTY(HOCHDRUCK, 0x07a6, Type::et_cent_val); 
+    PROPERTY(NIEDERDRUCK, 0x07a7, Type::et_cent_val); 
```

Comment:
Do you actually look at these values? What do they tell you? I just want to avoid adding all the available sensors for the sake of having them.

Reply:
I think a fundamental decision needs to be made. Do you want to only include and display the values that interest you, or do you want to enable everything that the heat pump offers? Personally, I prefer the second approach and have solved it by initially including everything in the code but setting most values in HA to ‘disabled_by_default’. This way, I can select and display the values that interest me in HA. Mainly, these are statistical values.

Reply:
What I want is a clean base that makes sense. This means we start small and add a bunch of entities that belong together, e.g. HOCHDRUCK NIEDERDRUCK, at the same time. But we do it end to end and test it.

The PRs are getting out of hand, they are too big, touch too many things and are sometimes not tested or even compiled. This way we won't get anywhere. Also you should fix the repo setup on your side to not having to manually copy paste stuff around. This way we can quickly and incrementally add stuff with small reviews.

With regards to new entities ... long term goal can be to have a full HA replacement for the WP display, I'm fine with that. This will be a long way, but is doable with a proper foundation. This was lacking in all other impls and people ended up with 10000 lines of yaml which was unmaintainable and took me a year to clean up. So I will definitely prevent it from going into that direction